src/acceptance-test/java/org/zalando/nakadi/domain/StreamMetadata.java

```diff
@@ -0,0 +1,42 @@
+package org.zalando.nakadi.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.Nullable;
+
+public class StreamMetadata {
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        final StreamMetadata that = (StreamMetadata) o;
+
+        return debug != null ? debug.equals(that.debug) : that.debug == null;
+
+    }
+
+    @Override
+    public int hashCode() {
+        return debug != null ? debug.hashCode() : 0;
+    }
+
+    public StreamMetadata(@Nullable @JsonProperty("debug") final String debug) {
+        this.debug = debug;
+    }
+
+    public String getDebug() {
```

Comment:
@rcillo I think if we are using this kind of annotation, than it's not enough.
Whenever I call method, I do not think about the way how the value was built.